go/nbs/table_reader.go

```diff
@@ -219,7 +219,9 @@ func (tr tableReader) get(h addr, stats *Stats) (data []byte) {
 	n, err := tr.r.ReadAt(buff, int64(offset))
 	stats.BytesPerRead.Sample(length)
 	stats.ChunksPerRead.SampleLen(1)
-	stats.ReadLatency.SampleTime(time.Since(t1))
+	if latency := time.Since(t1); latency >= 0 {
```

Comment:
lets be consistent with PutLatency, here and elsewhere. If time is 0 nanoseconds, let's just round it to 1.